examples/codeigniter/app/Views/errors/html/debug.js

```
@@ -0,0 +1,116 @@
+        if (tabListItems[i].nodeName == "LI")
+        {
+            var tabLink     = getFirstChildWithTagName(tabListItems[i], 'A');
+            var id          = getHash(tabLink.getAttribute('href'));
+            tabLinks[id]    = tabLink;
+            contentDivs[id] = document.getElementById(id);
+        }
+    }
+
+    // Assign onclick events to the tab links, and
+    // highlight the first tab
+    var i = 0;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)